B9PartSwitch/PartSwitch/PartModifiers/ModuleDataHandlerBasic.cs

```diff
@@ -34,16 +34,54 @@ public ModuleDataHandlerBasic(PartModule module, ConfigNode originalNode, Config
         public override void OnWillBeCopiedActiveSubtype() => Deactivate();
         public override void OnWasCopiedActiveSubtype() => Activate();
 
-        private void Activate()
+        protected virtual void Activate()
         {
             module.Load(dataNode);
             module.Events.Send("ModuleDataChanged", moduleDataChangedEventDetails);
         }
 
-        private void Deactivate()
+        protected virtual void Deactivate()
         {
             module.Load(originalNode);
             module.Events.Send("ModuleDataChanged", moduleDataChangedEventDetails);
         }
     }
+
+    public class ModuleFuelTanksHandler : ModuleDataHandlerBasic
```

Comment:
This should be in its own file

Reply:
Also can we get rid of the inheritance?  It doesn't really use any of the base behavior other than the list of events it responds to which is better just copied.

Reply:
Err to clarify, it should inherit from `PartModifierBase` rather than `ModuleDataHandlerBasic`